server/jest-controller/run-test.js

```diff
@@ -13,7 +13,7 @@ module.exports = async (domain, test) => {
     );
     let jsonStr = "";
     jest.stdout.on("data", (data) => {
-      jsonStr += data.toString();
+      jsonStr += data.toString().replace(/\\u001b\[[0-9]{1,2}m/g, "");
```

Comment:
Do we not need string quotations around this regex?

Reply:
In JS regexes are a first class concept.  You can parse it from a string. `const re = new RegExp('/whatev/g')` is equivalent to a raw `const re = /whatev/g`